src/dlghidden.h

```diff
@@ -26,8 +27,10 @@ class DlgHidden : public QWidget, public Ui::DlgHidden, public LibraryView {
     void selectAll();
     void selectionChanged(const QItemSelection&, const QItemSelection&);
 
+  private slots:
+    void slotActivateButtons(bool enable);
```

Comment:
is there a reason to have this (private) slotted? Could be a private member as it was?!